.travis.yml

```diff
@@ -43,10 +43,11 @@ install:
     - source ${PULUMI_SCRIPTS}/ci/install-common-toolchain.sh
 
     # Install Helm CLI. Do not install Tiller.
-    - curl -LO  https://storage.googleapis.com/kubernetes-helm/helm-v2.14.3-linux-amd64.tar.gz
-    - tar -xvf helm-v2.14.3-linux-amd64.tar.gz
+    - curl -LO  https://get.helm.sh/helm-v3.0.0-linux-amd64.tar.gz
+    - tar -xvf helm-v3.0.0-linux-amd64.tar.gz
```

Comment:
Since there's still Helm 2 in the world, should we continue to test both side-by-side for a period of time?

Reply:
I think it should be safe to just test v3 for now. v2 support is winding down over the next year, so I wouldn't expect a lot of changes.

I think testing both is more overhead than it's worth for now, but we can reevaluate if we get bug reports later.